build-rpi-raspbian/webif.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# start:     	starts a minimalistic web server
+# 	    	shows the status of OpenHABianPi installation
+# restart:   	checks if webserver is running 
+# inst_done: 	create finish message and link to http://$hostname:8080
+# cleanup:	stops the webserver
+#	     	removes all no longer needed files
+
+hostname=openhabianpi
+port=8888 # Port the webserver is listing to
+
+if [ $1 = "start" ]; then
+        mkdir /tmp/webif
```

Comment:
Please reindent the whole script

Reply:
sorry i don't understand what you mean

Reply:
Some lines are indented with 8 or 9 spaces.
The project style is two spaces ;) I know those are negligible details 🙃